apps/wasm/mvp/src/lib.rs

```diff
@@ -17,56 +20,53 @@ use libp2p::{
     Multiaddr, NetworkBehaviour,
 };
 use mina_network::p2p::{TransportBuilder, MAINNET_CONFIG};
+use pb::{requests::ConnectRequest, responses::CommonResponse};
 use std::{io, time::Duration};
-use wasm_bindgen::prelude::*;
+use utils::*;
 
 const TIMEOUT_SECS: u64 = 60;
 
 static mut EVENT_EMITTER: Option<EventEmitter> = None;
```

Comment:
Instead of trying to initialize this using unsafe blocks ourselves, it'd be good if we can use something like once_cell: https://docs.rs/once_cell/latest/once_cell/ Thoughts?

Reply:
The js ffi type does not meet the generic constraint of `OnceCell`

```
`*mut u8` cannot be shared between threads safely
within `js_ffi::EventEmitter`, the trait `Sync` is not implemented for `*mut u8`
required because of the requirements on the impl of `Sync` for `once_cell::imp::OnceCell<js_ffi::EventEmitter>`
shared static variables must have a type that implements `Sync`
```

Reply:
I've moved all `unsafe` code into a separate `event_emitter` mod and added `FIXME`.